POS-system-frontend/src/pages/TrialSignup.jsx

```
@@ -159,29 +164,44 @@ const TrialSignup = () => {
 
             {/* Terms Agreement */}
             <div className="mb-4 text-xs text-muted-foreground">
-              By signing up, you agree to our <a href="#" className="underline hover:text-primary">Terms</a> and <a href="#" className="underline hover:text-primary">Privacy Policy</a>.
+              By signing up, you agree to our <a href="#" className="underline hover:text-primary">Terms</a> and <a
```

Biome (2.3.13)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)